source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -389,7 +374,7 @@ void BioXASAppController::createGeneralPanes()
 	QWidget* beamlineConfigurationView = createComponentView(BioXASBeamline::bioXAS());
 	addMainWindowView( beamlineConfigurationView, "Configuration", generalPaneCategeryName_, generalPaneIcon_);
 
-	beamlineStatusView_ = new CLSBeamlineStatusView(BioXASBeamline::bioXAS()->beamStatus(), false);
+	beamlineStatusView_ = new CLSBeamlineStatusView(BioXASBeamline::bioXAS(), false);
```

Comment:
it is defined in the CLSAppController. This instantiation here might be able to moved to CLSAppController when we are having more beamlines using the CLSBeamStatusView